apps/src/feedback.js

```diff
@@ -972,6 +972,8 @@ FeedbackUtils.prototype.showClearPuzzleConfirmation = function(Dialog, hideIcon,
  * @param {object} options Configurable options.
  * @param {string} headerText Text for header portion
  * @param {string} bodyText Text for body portion
+ * @param {boolean} prompt Whether to prompt for a string value
+ * @param {string} promptPrefill If prompting, textbox prefill value
```

Comment:
Huh, looks like the JSDoc that was already here was mostly not quite correct. Could you update both your new additions and the existing params to properly document [parameters with properties](http://usejsdoc.org/tags-param.html#parameters-with-properties)?